meinberlin/react/livequestions/CategorySelect.jsx

```diff
@@ -21,7 +21,7 @@ export const CategorySelect = (props) => {
           <select
             name="categorySelect"
             id="categorySelect"
-            className="form-select"
+            className="form-control"
```

Comment:
Adding `form-control` to `CategorySelect` correctly styles the dropdown arrow:

<img width="973" alt="Screenshot 2024-11-11 at 15 30 46" src="https://github.com/user-attachments/assets/14fe6f46-bc12-4d4f-8b4c-35f717615989">